config/locales/js-en.yml

```diff
@@ -474,6 +474,7 @@ en:
     label_created_on: "created on"
     label_edit_comment: "Edit this comment"
     label_edit_status: "Edit the status of the work package"
+    label_edit_project_status: "Edit the status of the project"
```

Comment:
This should be part of the normal `en.yml` file. The `js-en.yml` should ideally only contain strings that need to be accessed from Angular.